<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>커넥션누수 on MIN Velog</title>
    <link>https://mintaewon.github.io/tags/%EC%BB%A4%EB%84%A5%EC%85%98%EB%88%84%EC%88%98/</link>
    <description>Recent content in 커넥션누수 on MIN Velog</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>ko-kr</language>
    <lastBuildDate>Sat, 14 Jan 2023 13:35:35 +0900</lastBuildDate><atom:link href="https://mintaewon.github.io/tags/%EC%BB%A4%EB%84%A5%EC%85%98%EB%88%84%EC%88%98/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>pymysql.err.InterfaceError: (0, &#39;&#39;) 에러</title>
      <link>https://mintaewon.github.io/2023/01/pymysql.err.interfaceerror-0-%EC%97%90%EB%9F%AC/</link>
      <pubDate>Sat, 14 Jan 2023 13:35:35 +0900</pubDate>
      
      <guid>https://mintaewon.github.io/2023/01/pymysql.err.interfaceerror-0-%EC%97%90%EB%9F%AC/</guid>
      <description>에러 : pymysql.err.InterfaceError: (0, &amp;lsquo;&amp;rsquo;) Python 서버로 만든 웹을 배포하고 이틀이지나자 갑자기 에러가 발생했다.
DB에 접속은 잘되는걸 보니 DB 문제는 아니었다.
서버의 로그를 보니
python에서 mysql과 연결을 도와주는 라이브러리는 pymysql의 오류였다.
cursors 와 connections 에 문제가 생긴 것 같다.
문제는 DB와 통신하는 코드에 있었다.
위 코드에서 conn은 DB 관련 파라미터를 지정해 연결을 하는 작업이고
curs는 SQl을 사용할 수 있게 DB에 접속을 한다는 것으로 이해했다.
그 후에 SQL 쿼리를 이용해 명령어를 사용하고</description>
    </item>
    
  </channel>
</rss>
